src/doc/man/rustdoc.1

```diff
@@ -1,4 +1,4 @@
-.TH RUSTDOC "1" "<INSERT DATE HERE>" "rustdoc <INSERT VERSION HERE>" "User Commands"
+.TH RUSTDOC "1" "July 2018" "rustdoc <INSERT VERSION HERE>" "User Commands"
```

Comment:
I pulled both of these dates from git log -- I think their non-recency in some sense is *good* as it emphasizes that we're not keeping these manpages in sync with rustc's CLI args. If that support is ever added, it'll probably be a more hefty process anyway, and maybe then including a date will make more sense.

Reply:
Yep, if someone cares about getting the manpages updated we should push for automatically generate them, and that tool could then depend on the `time` crate.